micropub.php

```diff
@@ -810,31 +810,44 @@ private static function check_error( $result ) {
 			return $result;
 		}
 
+
+		public static function get_micropub_endpoint() {
+			return apply_filters( 'micropub_endpoint', site_url( '?micropub=endpoint' ) );
```

Comment:
three new filters! cool! can you please add them to `README.md` and `readme.txt`, in both the _Filters and hooks_ sections and the changelogs?